mindmeld/app_manager.py

```diff
@@ -117,9 +117,9 @@ def load(self):
 
     def _pre_dm(self, processed_query, context, params, frame, history):
         request = self.request_class(context=context, history=history, frame=frame,
-                                     params=FrozenParams(previous_params=params), **processed_query)
+                                     params=params, **processed_query)
```

Comment:
We are assuming that params is a frozen params object here?

Reply:
The params to this function passes in a frozen params:

```
        params = freeze_params(params)
.
.
.

        request, response = self._pre_dm(processed_query=processed_query,
                                         context=context, history=history,
                                         frame=frame, params=params)
```